### ISSUE TITLE
Unable to compile

### ISSUE THREAD

**Issue by thomas41587** [absalom-muc/MHI-AC-Ctrl#34]
Hello,
I am trying to compile the project. I already added the reqired libraries but I get a bunch of error with the pubsub library. I checked the library itself and was able to compile the built in examples. So I guess the library itself should be OK.
Here is the output of the Arduino IDE: https://pastebin.com/0Q3QUWcs

**Comment by absalom-muc** [comment]
Hello Thomas,
I just compiled the [recently released version](https://github.com/absalom-muc/MHI-AC-Ctrl/releases/tag/v2.03) without problems. You could compare the versions of the used libraries:
![grafik](https://user-images.githubusercontent.com/23119513/88449613-860a2380-ce48-11ea-8518-fe8160e6796c.png)

You find in the web many posts related to ESP8266 compile errors. It seems to be not related to MHI-AC-Ctrl.

**Comment by thomas41587** [comment]
Thank you very much for the fast reply. I played a little bit around with the Arduino IDE but always ended up with the same error. Therefore I tried on a different VM-Image where nothing was install. Did the same steps there and...it compiled without any errors :-)
I have no idea what exactly caused these compile error. looks like something with the system itself seemed to be strange.
But now everything is working perfectly! Thank you very much for this awesome work!!
